$DragonFly: src/usr.sbin/ntpd/patches/Attic/ntp_msg.c.patch,v 1.2 2004/11/09 21:29:34 joerg Exp $

--- ntp_msg.c.orig	2004-11-09 22:18:03.000000000 +0100
+++ ntp_msg.c	2004-11-09 22:18:28.000000000 +0100
@@ -74,7 +74,7 @@
 
 int
 ntp_sendmsg(int fd, struct sockaddr *sa, struct ntp_msg *msg, ssize_t len,
-    int auth)
+    int auth __unused)
 {
 	char		 buf[NTP_MSGSIZE];
 	char		*p;
@@ -117,7 +117,7 @@
 	p += sizeof(msg->xmttime.fraction);
 
 	if (sa != NULL)
-		sa_len = SA_LEN(sa);
+		sa_len = sa->sa_len;
 	else
 		sa_len = 0;
 
